src/main/drivers/compass/compass_ak8975.c

```diff
@@ -59,29 +62,31 @@
 #define AK8975A_ASAY 0x11 // Fuse ROM y-axis sensitivity adjustment value
 #define AK8975A_ASAZ 0x12 // Fuse ROM z-axis sensitivity adjustment value
 
-static bool ak8975Init(void)
+static bool ak8975Init(magDev_t *magdev)
```

Comment:
Why not make this `static bool ak8975Init(busDevice_t *busDev)` since nothing in `magDev_t` seems to be used.

Reply:
Mhh, this is more consistent with the usage of the other sensor functions? We can change if you prefer.

Reply:
It's consistent with the others, but I think the others are wrong...
I made a start on changing the other drivers as part of the dual gyro support, but this work is incomplete. So mag be implemented to use `busDevice_t *` and the others will eventually do so as well.

And definitely NO local static pointer - one of the things we want to achieve is making all the drivers re-entrant.